fastlane/lib/fastlane/actions/ensure_git_status_clean.rb

```diff
@@ -65,7 +71,13 @@ def self.available_options
                                        description: "The flag whether to show the git diff if the repo is dirty",
                                        optional: true,
                                        default_value: false,
-                                       is_string: false)
+                                       is_string: false),
+          FastlaneCore::ConfigItem.new(key: :ignored,
+                                       env_name: "FL_ENSURE_GIT_STATUS_CLEAN_IGNORED_FILE",
+                                       description: "The flag whether to ignore file the git status if the repo is dirty",
+                                       optional: true,
+                                       default_value: false,
```

Comment:
Can we remove this `default_value`? I don’t think this needed since `optional: true` is set!

Once this is fixed we can get this merged! Thanks 😊